components/validation/src/main/java/org/wso2/carbon/identity/x509Certificate/validation/constant/error/ErrorMessage.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.x509Certificate.validation.constant.error;
+
+/**
+ * Error messages.
+ */
+public enum ErrorMessage {
+
+    // Client errors.
+    ERROR_INVALID_VALIDATOR_NAME("60001", "Invalid validator name.",
+            "Invalid validator name %s in the tenant %s."),
+    ERROR_NO_VALIDATORS_CONFIGURED_ON_TENANT("60002", "Unable to perform the operation.",
+            "No validator is configured on the given tenant %s."),
+    ERROR_NO_CA_CERTIFICATES_CONFIGURED_ON_TENANT("60003", "Unable to perform the operation.",
+            "No CA Certificate is configured on the given tenant %s."),
+    ERROR_CERTIFICATE_DOES_NOT_EXIST("60004", "Unable to perform the operation.", "Certificate " +
+            "with the id: %s does not exist in tenant %s."),
+
+    // Server errors.
+    ERROR_WHILE_RETRIEVING_VALIDATORS("65001", "Error while retrieving validators.",
+            "Error while retrieving validators from the system."),
+    ERROR_WHILE_RETRIEVING_VALIDATOR_BY_NAME("65002", "Error while retrieving validator by name.",
+            "Error while retrieving validator from the system."),
```

Comment:
include the name in the error description

Reply:
Address via https://github.com/wso2/product-is/issues/22818